templates/store/categories/iot.html

```diff
@@ -0,0 +1,456 @@
+              </p>
+            </div>
+          </div>
+        </a>
+      </div>
+    </div>
+
+    <div class="row is-bordered">
+      <h3 class="p-heading--four">Board images</h3>
+    </div>
+    <div class="row p-snap-row">
+      <div class="col-2">
+        <a class="p-media-object--snap" href="/pi2">
+          <img src="https://dashboard.snapcraft.io/site_media/appmedia/2015/04/berry.jpg.png" class="p-media-object--snap__img" alt="">
+          <div class="p-media-object--snap__content">
+            <h4 class="p-media-object--snap__title">
+              pi2
+            </h4>
+            <div class="p-media-object--snap__desc">
+              <p>
+                <span class="u-off-screen">Publisher: </span>Canonical
+                <span class="p-verified p-tooltip p-tooltip--top-center" aria-describedby="vlc-tooltip">
+                  <img src="https://assets.ubuntu.com/v1/75654c90-rosette.svg">
+                  <span class="p-tooltip__message u-align--center" role="tooltip" id="vlc-tooltip">Verified account</span>
+                </span>
+              </p>
+            </div>
+          </div>
+        </a>
+      </div>
+      <div class="col-2">
+        <a class="p-media-object--snap" href="/dragonboard">
+          <img src="https://dashboard.snapcraft.io/site_media/appmedia/2016/07/icon_32.png" class="p-media-object--snap__img" alt="">
+          <div class="p-media-object--snap__content">
+            <h4 class="p-media-object--snap__title">
+              dragonboard
+            </h4>
+            <div class="p-media-object--snap__desc">
+              <p>
+                <span class="u-off-screen">Publisher: </span>Canonical
+                <span class="p-verified p-tooltip p-tooltip--top-center" aria-describedby="vlc-tooltip">
+                  <img src="https://assets.ubuntu.com/v1/75654c90-rosette.svg">
+                  <span class="p-tooltip__message u-align--center" role="tooltip" id="vlc-tooltip">Verified account</span>
+                </span>
+              </p>
+            </div>
+          </div>
+        </a>
+      </div>
+      <div class="col-2">
+        <span class="p-media-object--snap">
+          <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" class="p-media-object--snap__img" alt="">
+          <div class="p-media-object--snap__content">
+            <h4 class="p-media-object--snap__title">
+              PC
+            </h4>
+            <div class="p-media-object--snap__desc">
+              <p>
+                <span class="u-off-screen">Publisher: </span>SORACOM Snap Administrator
+              </p>
+            </div>
+          </div>
+        </span>
+      </div>
+    </div>
+
+    <div class="row is-bordered">
+      <h3 class="p-heading--four">More on IoT & Snapcraft</h3>
+    </div>
+    <div class="row">
+      <div class="u-equal-height">
+        <div class="col-4 p-card--post">
+          <div class="p-card__content">
```

Comment:
This `.p-card__content` seems to be having unnecessary top dotted border.

<img width="990" alt="screen shot 2018-08-09 at 12 49 46" src="https://user-images.githubusercontent.com/83575/43894723-41593b62-9bd3-11e8-871a-209ca2fcc783.png">

Possibly it's a vanilla issue? 
<img width="595" alt="screen shot 2018-08-09 at 12 50 38" src="https://user-images.githubusercontent.com/83575/43894722-413f2eb6-9bd3-11e8-9130-295ef614f330.png">

Reply:
I just noticed that we have the same issue on `/blog`:

<img width="806" alt="screen shot 2018-08-09 at 12 56 29" src="https://user-images.githubusercontent.com/83575/43894878-c6f7adbc-9bd3-11e8-8120-155a2bdaedf2.png">

Reply:
Reported separately: https://github.com/canonical-websites/snapcraft.io/issues/1004